azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -989,7 +989,7 @@ private class CleanerThread extends Thread {
     // Every 5 mins kill flows running longer than allowed max running time
     private static final long LONG_RUNNING_FLOW_KILLING_INTERVAL_MS = 5 * 60 * 1000;
     private final long flowMaxRunningTimeInMins = FlowRunnerManager.this.azkabanProps.getInt(
-        Constants.ConfigurationKeys.AZKABAN_MAX_FLOW_RUNNING_MINS, -1);
+        Constants.ConfigurationKeys.AZKABAN_MAX_FLOW_RUNNING_MINS, 10 * 24 * 60); // 10 days
```

Comment:
Thank you for fixing this bug. Can you please create constant for 10 days value? DEFAULT_AZKABAN_MAX_FLOW_RUNNING_MINS?

Reply:
Thx! Addressed.